public/app/features/alerting/unified/Analytics.ts

```diff
@@ -39,3 +39,16 @@ export function withPerformanceLogging<TFunc extends (...args: any[]) => Promise
     return response;
   };
 }
+
+export const trackAlertRuleCreation = (props: AlertRuleTrackingProps) => {
```

Comment:
What do you think about a bit more user-action-oriented names like `trackNewAlerRuleFormOpened`

Reply:
Sounds good, changed both names to `trackNewAlerRuleFormSaved` and `trackNewAlerRuleFormCancelled`. I used the word `saved` instead of `opened` because it's actually tracking after a successful save, not just when it opens.